### PR TITLE
[Sample/Chat&Call] adding scope param for token request

### PR DESCRIPTION
**Pull request by carocao-msft**
# What
For calling and chat ,added ?scope=chat so only the corresponding token will be minted 
Related task: https://skype.visualstudio.com/SPOOL/_workitems/edit/2691358

### CODE REVIEW

**Review by JamesBurnside — APPROVED** [review]

File: samples/Calling/src/app/utils/AppUtils.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { v1 as generateGUID } from 'uuid';
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const fetchTokenResponse = async (): Promise<any> => {
-  const response = await fetch('/token');
+  const response = await fetch('/token?scope=voip');
```
Comment (PorterNan) [inline samples/Calling/src/app/utils/AppUtils.ts:12]: Question about the PR title, this is a client-side change right? And why are we removing scope from client side request?

Comment (PorterNan) [reply samples/Calling/src/app/utils/AppUtils.ts:12]: Maybe like [Sample/Chat&Call] adding scope param for token request

**Review by PorterNan — APPROVED** [review]

File: samples/Calling/src/app/utils/AppUtils.ts (hunk 1 above)
Comment (PorterNan) [reply samples/Calling/src/app/utils/AppUtils.ts:12]: Then add the related PR of server side in the description, so people can check then together when reviewing

Comment (carocao-msft) [reply samples/Calling/src/app/utils/AppUtils.ts:12]: Updated :)

**Pull request merged**